proposals/dl_fit_class_mlp.json

```diff
@@ -0,0 +1,159 @@
+{
+    "id": "dl_fit_class_mlp",
```

Comment:
what is that `dl_` prefix?

Reply:
meant to show deep learning but ml will do.